test/unit/puppetlabs/services/ca/certificate_authority_core_test.clj

```diff
@@ -99,59 +102,63 @@
-                   "DNS:revoked-agent"]
+  {:dns_alt_names     ["DNS:BAR"
+                       "DNS:Baz4"
+                       "DNS:foo"
+                       "DNS:revoked-agent"]
    :subject_alt_names ["DNS:BAR"
                        "DNS:Baz4"
                        "DNS:foo"
                        "DNS:revoked-agent"]
-   :fingerprint   "1C:D0:29:04:9B:49:F5:ED:AB:E9:85:CC:D9:6F:20:E1:7F:84:06:8A:1D:37:19:ED:EA:24:66:C6:6E:D4:6D:95"
-   :fingerprints  {:SHA1 "38:56:67:FF:20:91:0E:85:C4:DF:CA:16:77:60:D2:BB:FB:DF:68:BB"
-                   :SHA256 "1C:D0:29:04:9B:49:F5:ED:AB:E9:85:CC:D9:6F:20:E1:7F:84:06:8A:1D:37:19:ED:EA:24:66:C6:6E:D4:6D:95"
-                   :SHA512 "1A:E3:12:14:81:50:38:19:3C:C6:42:4B:BB:09:16:0C:B1:8A:3C:EB:8C:64:9C:88:46:C6:7E:35:5E:11:0C:7A:CC:B2:47:A2:EB:57:63:5C:48:68:22:57:62:A1:46:64:B4:56:29:47:A5:46:F4:BD:9B:45:77:19:91:0B:35:39"
-                   :default "1C:D0:29:04:9B:49:F5:ED:AB:E9:85:CC:D9:6F:20:E1:7F:84:06:8A:1D:37:19:ED:EA:24:66:C6:6E:D4:6D:95"}
-   :name          "revoked-agent"
-   :state         "revoked"})
+   :fingerprint       "1C:D0:29:04:9B:49:F5:ED:AB:E9:85:CC:D9:6F:20:E1:7F:84:06:8A:1D:37:19:ED:EA:24:66:C6:6E:D4:6D:95"
+   :fingerprints      {:SHA1    "38:56:67:FF:20:91:0E:85:C4:DF:CA:16:77:60:D2:BB:FB:DF:68:BB"
+                       :SHA256  "1C:D0:29:04:9B:49:F5:ED:AB:E9:85:CC:D9:6F:20:E1:7F:84:06:8A:1D:37:19:ED:EA:24:66:C6:6E:D4:6D:95"
+                       :SHA512  "1A:E3:12:14:81:50:38:19:3C:C6:42:4B:BB:09:16:0C:B1:8A:3C:EB:8C:64:9C:88:46:C6:7E:35:5E:11:0C:7A:CC:B2:47:A2:EB:57:63:5C:48:68:22:57:62:A1:46:64:B4:56:29:47:A5:46:F4:BD:9B:45:77:19:91:0B:35:39"
+                       :default "1C:D0:29:04:9B:49:F5:ED:AB:E9:85:CC:D9:6F:20:E1:7F:84:06:8A:1D:37:19:ED:EA:24:66:C6:6E:D4:6D:95"}
+   :name              "revoked-agent"
+   :state             "revoked"
+
+   :not_after     "2019-08-25T19:25:29UTC"
+   :not_before    "2014-08-25T19:25:29UTC"
+   :serial_number 4})
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;; Tests
 
 (deftest crl-endpoint-test
   (testing "implementation of the CRL endpoint with no 'If-Modified-Since' header"
-    (let [request (mock/request :get
-                            "/v1/certificate_revocation_list/mynode")
+    (let [request  (mock/request :get
+                                 "/v1/certificate_revocation_list/mynode")
           response (handle-get-certificate-revocation-list
-                     request {:cacrl (test-pem-file "crl.pem") :infra-crl-path (test-pem-file "crl.pem") :enable-infra-crl false})]
+                    request {:cacrl (test-pem-file "crl.pem") :infra-crl-path (test-pem-file "crl.pem") :enable-infra-crl false})]
       (is (map? response))
       (is (= 200 (:status response)))
       (is (= "text/plain" (get-in response [:headers "Content-Type"])))
       (is (string? (:body response))))))
-  (testing "with a malformed http-date 'If-Modified-Since' header"
```

Comment:
This alignment shouldn't be like this. It looks like your editor is doing it this way because the parens are off. I'm certain these `testing` forms should be within `deftest crl-endpoint-test`.

Reply:
You're right. I'll push a rebased commit in a sec.